### PR TITLE
Add missing morph expressions and their images to docs.

### LINKED ISSUE #1553: Add missing region and locset expressions to the s-expression parser and docs.
Missing from the s-expression parser: 
- `reg::complement`
- `reg::difference`
- `ls::boundary`
- `ls::cboundary`
- `ls::segment_boundaries`
- `ls::support`

Missing from docs: 
- `reg::complement`
- `reg::difference`
- `reg::z_dist_from_root_*`
- `reg::complete`
- `ls::boundary`
- `ls::cboundary`
- `ls::segment_boundaries`
- `ls::support`
- `ls::on_components`

### PR DESCRIPTION
**Pull request by thorstenhater**
Document missing morph DSL expressions.

Closes #1553

### CODE REVIEW

**Review by brenthuisman — CHANGES_REQUESTED** [review]
If you add `segment_boundaries`, I think you'd close #1553.

File: doc/concepts/labels.rst [hunk 1]
```diff
@@ -297,9 +307,18 @@ Locset expressions
...
+    ``(boundary (segment 2))``, ``(cboundary (segment 2))``, and ``(segment-boundaries)``.
+
+    .. figure:: ../gen-images/boundaries_label.svg
+      :width: 600
+      :align: center
+
 .. label:: (proximal-translate ls:locset distance:real)
 
-    The set of locations that correspond to moving each location in the ``ls`` in the proximal direction
+    The set of locations that correspond to moving each location in the ```ls`` in the proximal direction
```
Comment (brenthuisman) [inline doc/concepts/labels.rst:321]: Why the extra tick?

File: doc/scripts/inputs.py
Comment (brenthuisman) [inline doc/scripts/inputs.py:1]: Is this only formatting or more?

File: doc/concepts/labels.rst (hunk 1 above)
Comment (thorstenhater) [reply doc/concepts/labels.rst:321]: Auto fill

File: doc/scripts/inputs.py
Comment (thorstenhater) [reply doc/scripts/inputs.py:1]: This is adds the coordinates spit out by `gen-labels.py` as someone had manually edited this file to pleasse black/flake, it is all different now.

Comment (thorstenhater) [reply doc/scripts/inputs.py:1]: and the iexpr data was added by hand.

**Comment by thorstenhater** [comment]
> If you add `segment_boundaries`, I think you'd close #1553.

I did, see `boundary`, `cboundary`, and `segment_boundaries`.

**Comment by brenthuisman** [comment]
Ah, lazy CTRL-F on my part.

**Review by brenthuisman — APPROVED** [review]

**Pull request merged**